src/AddHavingAggregateTypesPlugin.ts

```diff
@@ -595,12 +597,13 @@ const Plugin: GraphileConfig.Plugin = {
             case "float":
             case "bigfloat":
             case "datetime": {
-              addBinaryOp("equalTo", sql.fragment`=`);
-              addBinaryOp("notEqualTo", sql.fragment`<>`);
-              addBinaryOp("greaterThan", sql.fragment`>`);
-              addBinaryOp("greaterThanOrEqualTo", sql.fragment`>=`);
-              addBinaryOp("lessThan", sql.fragment`<`);
-              addBinaryOp("lessThanOrEqualTo", sql.fragment`<=`);
+              // TODO: WTF?
+              // addBinaryOp("equalTo", sql.fragment`=`);
+              // addBinaryOp("notEqualTo", sql.fragment`<>`);
+              // addBinaryOp("greaterThan", sql.fragment`>`);
+              // addBinaryOp("greaterThanOrEqualTo", sql.fragment`>=`);
+              // addBinaryOp("lessThan", sql.fragment`<`);
+              // addBinaryOp("lessThanOrEqualTo", sql.fragment`<=`);
```

Comment:
@benjie Not quite sure how to handle these cases. When it's uncommented, exporting gives me the following error:

```
        throw new Error(`Exporting of 'sql' values is not supported (at ${locationHint}), please wrap in EXPORTABLE: ${sql ? sql.compile(thing).text : (0, node_util_1.inspect)(thing)}`);
              ^

Error: Exporting of 'sql' values is not supported (at HavingDatetimeFilter.fields[equalTo].extensions.grafast["applyPlan"][$$scope][1]), please wrap in EXPORTABLE: =
    at _convertToAST (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1015:11)
    at convertToIdentifierViaAST (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1156:7)
    at <anonymous> (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1279:12)
    at Array.map (<anonymous>)
    at factoryAst (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1265:37)
    at convertToIdentifierViaAST (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1155:7)
    at handleSubvalue (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:983:19)
    at <anonymous> (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1050:24)
    at Array.forEach (<anonymous>)
    at _convertToAST (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1048:13)
    at convertToIdentifierViaAST (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1156:7)
    at <anonymous> (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1652:13)
    at Array.forEach (<anonymous>)
    at exportSchemaTypeDefs (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1536:15)
    at exportSchemaAsString (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1850:5)
    at exportSchema (/home/nick/code/cloud/tycho-service/packages/operations-metrics/node_modules/graphile-export/src/exportSchema.ts:1949:26)
    at main (/home/nick/code/cloud/tycho-service/packages/operations-metrics/tools/export-schema.ts:14:9)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at <anonymous> (/home/nick/code/cloud/tycho-service/packages/operations-metrics/tools/export-schema.ts:21:1)
```

and I'm not quite sure what to wrap in EXPORTABLE

Reply:
Probably the best bet is to change the `infix` parameter to `addBinaryOp` from `infix: SQL` to `infix: SQL | (() => SQL)`; then each of these would become:

```ts
addBinaryOp("equalTo", EXPORTABLE((sql) => sql.fragment`=`, [sql]));
```

Reply:
Had to change it to `infix: () => SQL` and execute the function instead, but it works!